testem.travis-browsers.js

```diff
@@ -1,4 +1,4 @@
-var FailureOnlyReporter = require('./lib/failure-only-testem-reporter');
+const FailureOnlyReporter = require('./lib/failure-only-testem-reporter');
```

Comment:
The only difference in this file and the other one seems to be the list of browsers, can we instead use a single `testem.js` along with some environment variable inspection to determine the browsers?

Reply:
yeah, that might be a good idea for the future